src/commands/createNewProject/ProjectCreateStep/PythonProjectCreateStep.ts

```diff
@@ -30,10 +33,34 @@ const defaultRequirements: string = `# DO NOT include azure-functions-worker in
 azure-functions
 `;
 
+const azureWebJobsFeatureFlagsKey = 'AzureWebJobsFeatureFlags';
+
+const defaultFunctionAppSource = `import azure.functions as func
```

Comment:
Is this just a placeholder until the Python language model templates are available or are we going to have to hardcode the source code like this?

Reply:
For this preview version, we're now bundling as "backup templates" so they're in the standard `templates.json` format in the `resources` folder.